ai/bert/Cargo.toml

```diff
@@ -22,9 +22,7 @@ tract-onnx = "0.18.3"
 onnxruntime = { version = "0.0.13", optional = true }
 
 [dev-dependencies]
-criterion = { workspace = true }
 csv = { workspace = true }
-indicatif = { workspace = true }
```

Comment:
❓:wrench: why is this removed? without it the `validate` example declared below fails to compile.

Reply:
I trusted `udeps` to tell me which deps where not used. I guess we also need to modify our just file/CI because it is not failing.

Reply:
I guess is my fault, I didn't use `--all-features`

Reply:
the ci doesn't check them because they have `required-features` which are not given in the ci action, `cargo check` just skips those. it should for example fail if we add an additional check with `--all-features` (which is also the reason why i see this locally on my machine).